### ISSUE TITLE
New Release `1.2.0`?

### ISSUE THREAD

**Issue by pcj** [mchome/flutter_colorpicker#121]
I'd like to get https://github.com/mchome/flutter_colorpicker/pull/111 without having to pin this commit in my pubspec.  Thanks!

**Comment by C0Florent** [comment]
I would need this too

**Comment by rinscr3003** [comment]
Yes I need this too. I've debugged for half an hour and finally found wrong codes in `utils.dart`. When I decided to make a PR I found this had already been solved in #111 but with dependency `flutter_colorpicker: ^1.1.0` I got this bug again this morning lol.
